Extras/FrontendTests/tests/keyboard.spec.ts

```diff
@@ -12,8 +12,8 @@ test('Test keyboard events', {
 }, async ({ page, streamerPage, streamerId }) => {
 
     await page.goto(`/?StreamerId=${streamerId}&MatchViewportRes=true`);
-    await page.getByText('Click to start').click();
-    await helpers.waitForVideo(page);
+
+    await helpers.startAndWaitForVideo(page);
```

Comment:
Previously where it would click and then call wait for video, sometimes it would execute the whole thing in the call to click and `playStream` event that the wait for video waits on would already be fired. This change ensures the listening to the event is done before it does the starting of the stream.